docs/install.html

```diff
@@ -116,15 +117,114 @@ <h2><a name="installation"></a>Installation</h2>
 	<p>
 		If it isn't already, make sure the cache directory has permissions of at least 0700 and is
 		owned by the process your webserver is running under. This is used to cache RSS files. It
-		is NOT recommended to set the directory to full write, 0777.
+		is <em>not</em> recommended to set the directory to full write, 0777.
 	</p>
 
+	<h3>Configuration</h3>
+
+	<p>
+		WebSVN always requires a special configuration file, for which a template is provided as
+		<code>include/distconfig.php</code>. Only the things mentioned in that template are valid
+		configs at all and some of those are even necessary, e.g., WebSVN obviously needs to know
+		which SVN repos it should publish. Making the necessary config file available to WebSVN can
+		be done using two ways: Either simply by copying the template file to the hard-coded path
+		<code>include/config.php</code> and/or by setting the environment variable
+		<code>WEBSVN_PATH_CONF</code> in a way so that PHP can read its value. That value is
+		expected to be the path to the config file to use and things like symlinks work as well of
+		course, as long as those are handled transparently by the file system.
+	</p>
+
+	<p>
+		Using the environment variable has two main benefits: One doesn't neet to store additional
+		files in the deployment of WebSVN and the path given by <code>WEBSVN_PATH_CONF</code> is
+		used <em>additionally</em> to <code>include/config.php</code>, if that file is present at
+		all. If not, the file given by the environment variable is used exclusively. This approach
+		allows to override specially chosen configs of some main configuration file using another
+		one based on things like the current location of a request to WebSVN. A webserver like
+		Apache HTTPd could be configured using <code>Location</code> blocks to provide different
+		paths to config files using the environment variable like in the following example:
+	</p>
+
+	<pre>
+&lt;Location "/websvn/[...]"&gt;
+	SetEnv WEBSVN_PATH_CONF "/home/websvn/config/[...]/config.php"
+&lt;/Location&gt;</pre>
+
+	<p>
+		Considering both the hard coded config file path and the more dynamic environment variable
```

Comment:
consistency: hardcoded

Reply:
If I search dict.leo.org, there's no `hardcoded`, but only a `hard-coded` instead:

https://dict.leo.org/englisch-deutsch/hardcoded

You have confused me, isn't `hard-coded` the correct adjective where ever I used it? `hard-coded` path, `hard-coded` config file... Pretty much like in the second example of the following:

> The developer hard coded the error messages for the program.
> Hard-coded strings are a bad idea.

https://developer.salesforce.com/docs/atlas.en-us.salesforce_pubs_style_guide.meta/salesforce_pubs_style_guide/style_hard_coding.htm

Reply:
Really my bad, you are right: https://www.lexico.com/en/definition/hard-code. Please leave it as-is, but make it consistent throughout the document.